src/eureka/S3_data_reduction/nircam.py

```diff
@@ -355,15 +370,19 @@ def corr_oneoverf(data, meta, i, star_pos_x, log):
         for j in range(128):
             for k in range(4):
                 if ampl_used_bool[k]:
-                    data.flux.values[i][j, edges_all[k][0]:edges_all[k][1]] -= \
-                        me.meanerr(flux_all[k][j], err_all[k][j], mask=mask_all[k][j], err=False)
+                    edges_temp = edges_all[k]
+                    data.flux.values[i][j, edges_temp[0]:edges_temp[1]] -= \
+                        me.meanerr(flux_all[k][j], err_all[k][j],
+                                   mask=mask_all[k][j], err=False)
     elif meta.oneoverf_corr == 'median':
         for k in range(4):
             if ampl_used_bool[0]:
-                data.flux.values[i][:, edges_all[k][0]:edges_all[k][1]] -= \
+                edges_temp = edges_all[k]
+                data.flux.values[i][:, edges_temp[0]:edges_temp[1]] -= \
```

Comment:
Hey @taylorbell57 , out of curiosity, any reason you changed 
data.flux.values[i][:, edges_all[k][0]:edges_all[k][1]] to 
edges_temp = edges_all[k] and then 
data.flux.values[i][:, edges_temp[0]:edges_temp[1]] ?

Reply:
Yeah, was just to make things fit better with the 80 character line limit. Without that, it was going to get really awkward to get it all compliant.